### DISCUSSION TITLE
Any documentaion/api for rate limits?

### DISCUSSION THREAD

**Discussion by suvam0451** [misskey-dev/misskey#14440] [General]
I am developing a Misskey mobile client and ran into **RATE_LIMIT_EXCEEDED** error

<img width="651" alt="Screenshot 2024-08-19 at 2 26 30 PM" src="https://github.com/user-attachments/assets/18218b56-1384-40b5-9315-3bc1e732fe93">

It has been a **couple hours already and I am still rate limited (as seen from web UI)**

Is there any documentation and/or api endpoints to obtain/show rate limit information client-side (about how much longer one has to wait, for example) ?

<img width="496" alt="Screenshot 2024-08-19 at 2 21 49 PM" src="https://github.com/user-attachments/assets/e6b37960-d461-49bc-ab51-de02c7d30e67">

**Comment by suvam0451** [comment]
The endpoint in question is **notes/favorites/create**, to be specific. Is this a bug?

**Comment by KisaragiEffective** [comment]
The Project would not support a specific instance of hosting server. In general, modifications on the instance may affect your possibility. Please contact Server's administrator support if you still believe that is an error.
There is hope that the instance may accept patch that [adds `Retry-After` header in the rate-limit response](https://github.com/misskey-dev/misskey/pull/13949), but The Project would not be able to say it is going to happen.
For a documentation on the Project side, please refer https://github.com/misskey-dev/misskey/issues/14494
